mrp_production_project_estimated_cost/models/mrp_production.py

```diff
@@ -15,19 +15,42 @@
 #    along with this program.  If not, see http://www.gnu.org/licenses/.
 #
 ##############################################################################
-from openerp import models, api, exceptions, _
+from openerp import models, fields, api, exceptions, _
 
 
 class MrpProduction(models.Model):
     _inherit = 'mrp.production'
 
-    @api.one
+    active = fields.Boolean('Active', default=False)
+    name = fields.Char(
+        string='Referencen', required=True, readonly=True, copy="False",
+        states={'draft': [('readonly', False)]}, default="/")
+
     @api.model
-    def action_confirm(self):
+    def create(self, values):
+        sequence_obj = self.pool['ir.sequence']
+        if values['active']:
+            values['name'] = sequence_obj.get(self._cr, self._uid,
+                                              'mrp.production')
+        else:
+            values['name'] = sequence_obj.get(self._cr, self._uid,
+                                              'fictitious.mrp.production')
+        production = super(MrpProduction, self).create(values)
+        return production
+
+    @api.multi
+    def action_compute(self, properties=None):
```

Comment:
Si si... @pedrobaeza está bien así.
Si calculas datos, estás cambiando todo, materiales, máquinas... "reiniciando" de nuevo la OF. Por tanto, es correcto que se recalcule el árbol de costes. En otro caso, no coincidiría. 

El botón "calcular estimaciones" o lo que sea, nos servirá para SOLO recalcular la estructura de costes, en caso de que cambien materiales, o máquinas... sin reiniciar todo. 
Estaría correcto así.